### PR TITLE
Bump to version 6.0.0-rc2

### PR DESCRIPTION
**Pull request by talevy**
docs version update will happen separately after release

### CODE REVIEW

File: docs/Versions.asciidoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-:version:               6.0.0-rc1
+:version:               6.0.0-rc2
```
Comment (jasontedor) [inline docs/Versions.asciidoc]: This should not be changed here now. It needs to stay RC1 until RC2 is released.

**Review by jasontedor — CHANGES_REQUESTED** [review]
I left a comment.

File: docs/Versions.asciidoc (hunk 1 above)
Comment (talevy) [reply docs/Versions.asciidoc]: Ah, right. docs have a slightly different cycle. will update

**Comment by talevy** [comment]
thanks for the quick feedback, I've reverted the doc version bump

**Comment by talevy** [comment]
jenkins, test this please

**Review by jasontedor — APPROVED** [review]
LGTM.

**Pull request merged**